Equilobe.DailyReport.Models/Storage/ReportSettings.cs

```diff
@@ -26,7 +27,7 @@ public class ReportSettings
 
         public string PolicyXml { get; set; }
 
-        // NO DB
-        //public Policy Policy { get; set; }
+        [NotMapped]
+        public JiraPolicy Policy { get; set; }
```

Comment:
proprietatea asta imi permite sa utilizez clasele actuale pentru a trimite date in UI, altfel crapa pentru ca nu sunt instantiate anumite proprietati [care nu imi folosesc si nu sunt folosite de fapt decat in cazuri stricte]
ORICUM, asta e doar temporar pana la refactor, iar NotMapped face exact ce trebuie sa nu fie integrat in db cand fac retrieve/save pentru a putea testa binding-urile